terraform_module/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "commands" {
+  type = list(string)
+  default = [
+    "echo 'Executed example1'",
+    "echo 'Executed example2'",
+    "echo 'Executed example3'"
+  ]
+}
```

Comment:
Small nitpick, a couple files are missing an line break at the end of file. This is mostly a conventional thing for posix compliance and doesn't matter in any functional way on modern systems. Good explanation [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)